gaia-header.js

```diff
@@ -643,6 +650,26 @@ module.exports = component.register('gaia-header', {
         if (value) { this.setAttr('no-font-fit', ''); }
         else { this.removeAttr('no-font-fit'); }
       }
+    },
+
+    // The [ignore-dir] attribute can be used to force the older behavior of
+    // gaia-header, where the whole header is always displayed in LTR mode.
+    ignoreDir: {
+      get: function() {
+        if ('ignoreDir' in this) { return this._ignoreDir || false; }
+        return this.hasAttribute('ignore-dir');
+      },
+      set: function(value) {
+        debug('set ignore-dir', value);
+        value = !!(value || value === '');
+
+        if (value === this.ignoreDir) { return; }
+        this._ignoreDir = value;
+        this.dirObserver &= !value;
```

Comment:
but what happens if the user set the attribute and then remove it ? But maybe we don't care about this.

also be careful that `&` and `&&` are different operators, I was bitten by this in the past. In this case I think we're fine because of the `!`.